internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,17 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	examplesJSON, err := prettyJSON([]byte(sb.String()))
+	if err != nil{
+		return err
+	}
+	csv.GetAnnotations()["alm-examples"] = examplesJSON
 	return nil
 }
+
+// prettyJSON returns a JSON in a pretty format
+func prettyJSON(b []byte) (string, error) {
+	var out bytes.Buffer
+	err := json.Indent(&out, b, "", "  ")
+	return out.String(), err
+}
```

Comment:
Nit: add a newline at the end of the last line in the file.

Reply:
👍 ok.